.github/bin/trufflehog

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+docker run --rm -v "$(pwd):/workdir" \
```

Comment:
The alternative was compiling for our specific system whereas with this we can install a 62 MB docker image in no time